.github/workflows/TestBinaryInstaller.yml

```diff
@@ -76,3 +76,12 @@ jobs:
           rm -rf vendor
           ddev composer install --no-cache
           test -x vendor/bin/task
+
+      - name: Prove we can upgrade task within task
+        run: |
+          cd drainpipe/tests/fixtures/drainpipe-task-upgrade
+          ddev start
+          ddev composer install
+          ddev task drupal:composer:development
+          cp composer-upgraded.lock composer.lock
+          ddev task drupal:composer:development
```

Comment:
This isn't working with our fixed code because we're still installing drainpipe from the last release, and not this branch. I need to fix that!